dotcom-rendering/playwright/tests/parallel-1/sign-in-gate.e2e.spec.ts

```diff
@@ -0,0 +1,404 @@
+import type { BrowserContext, Page } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+import { addCookie, clearCookie } from 'playwright/lib/cookies';
+import { getIframeBody } from 'playwright/lib/iframe';
+import { expectToBeVisible, expectToNotExist } from 'playwright/lib/locators';
+import { Labs as labsArticle } from '../../../fixtures/generated/articles/Labs';
+import { Standard as standardArticle } from '../../../fixtures/generated/articles/Standard';
+import {
+	COMPLETE_REGISTRATION_BUTTON,
+	SIGN_IN_INCENTIVES_DIGITAL,
+	SIGN_IN_PROMPT,
+	SUBSCRIPTION_HEADER,
+} from '../../../src/lib/signInAfterCheckOutText';
+import { CMP_LAYER1_IFRAME, cmpAcceptAll, disableCMP } from '../../lib/cmp';
+import { loadPageWithOverrides } from '../../lib/load-page';
+
+const setArticleCount = async (page: Page, dailyArticleCount: number) => {
+	await page.evaluate((n) => {
+		window.localStorage.setItem(
+			'gu.history.dailyArticleCount',
+			JSON.stringify({
+				value: [{ day: Math.floor(Date.now() / 86400000), count: n }],
+			}),
+		);
+	}, dailyArticleCount);
+};
+
+const setMvtCookie = async (context: BrowserContext) => {
+	// sign in gate main runs from 0-900000 MVT IDs, so 500 forces user into test
+	await addCookie(context, { name: 'GU_mvt_id', value: '500000' });
```

Comment:
What’s this comment about `500`?

```suggestion
	// sign in gate main runs from 0-900000 MVT IDs, so 500 forces user into test
	await addCookie(context, { name: 'GU_mvt_id', value: String(500_000) });
```

Reply:
Done. Looks like a typo in the old tests comments.